haskell/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:16.04
+# # Copy Wikipedia
+RUN apt-get update && apt-get install -y curl
+RUN mkdir -p /mnt/data
+RUN curl http://traildb.io/data/wikipedia-history-small.tdb > /mnt/data/wikipedia-history-small.tdb
```

Comment:
`ADD` will fetch the file automatically, i.e. https://github.com/joeyrobert/traildb-benchmark/blob/master/crystal/Dockerfile#L22

Reply:
But `curl` will make it use Docker cache and `ADD` will not use it reliably. Or maybe it's just me.